bot/exts/info/help.py

```diff
@@ -126,14 +126,7 @@ async def command_not_found(self, string: str) -> "HelpQueryNotFound":
         Will return an instance of the `HelpQueryNotFound` exception with the error message and possible matches.
         """
         choices = await self.get_all_help_choices()
-
-        # Run fuzzywuzzy's processor beforehand, and avoid matching if processed string is empty
-        # This avoids fuzzywuzzy from raising a warning on inputs with only non-alphanumeric characters
-        if (processed := full_process(string)):
-            result = process.extractBests(processed, choices, scorer=fuzz.ratio, score_cutoff=60, processor=None)
-        else:
-            result = []
-
+        result = process.extract(default_process(string), choices, scorer=fuzz.ratio, score_cutoff=60, processor=None)
```

Comment:
Shouldn't we check for the result of `default_process` here anymore, in case it returns an empty string? Or does it not matter as much anymore, since `rapidfuzz` won't issue a warning like `fuzzywuzzy`?

Reply:
That was basically my reasoning. The original comment said that was done due to the fuzzywuzzy warning, which doesn't exist in rapidfuzz, so it'll safely return an empty list.

Reply:
Yes, that's a great idea 👍